velox/common/caching/StringIdMap.h

```diff
@@ -63,6 +64,12 @@ class StringIdMap {
     return it == idToString_.end() ? "" : it->second.string;
   }
 
+  void testingOverflow() {
```

Comment:
```
void testingSetLastId(uint64_t lastId) {
  lastId_ = lastId;
}
```

Reply:
Hi @xiaoxmeng , I have changed the testing function, please take a look, thanks!